src/hoi4_world/countries/hoi4_country_converter_tests.cpp

```diff
@@ -1810,4 +1907,90 @@ TEST(Hoi4worldCountriesCountryConverter, StabilityConvertsFromLegitimacy)
    EXPECT_FLOAT_EQ(country_two->GetStability(), 0.00F);
 }
 
-}  // namespace hoi4
\ No newline at end of file
+TEST(Hoi4worldCountriesCountryConverter, UnitsAreConverted)
```

Comment:
There's a few edges cases resulting in missed lines in the coverage report: https://coveralls.io/builds/62251077/source?filename=src%2Fhoi4_world%2Fcountries%2Fhoi4_country_converter.cpp

If possible and reasonable, add test cases to catch those.

Reply:
Done.